### PR TITLE
Ceiling

### PR DESCRIPTION
**Pull request by M3RF3R**
### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Tags:
+  - 'Libraries'
+  - 'Arithmetic'
+  - 'Functions'
+Catalog Content:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+**Math.Celiling()** is a class method that always rounds up to the next full integer. It is used to return the smallest integer greater than or equal to the specified number.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md:16]:
```suggestion
**Math.Ceiling()** is a class method that always rounds up to the next full integer. It is used to return the smallest integer greater than or equal to the specified number.
```

File: content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## Syntax
+
+```pseudo
+public static decimal Ceiling (decimal val);
+public static double Ceiling(double val)
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md:22]:
```suggestion

```

File: content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+**Math.Celiling()** is a class method that always rounds up to the next full integer. It is used to return the smallest integer greater than or equal to the specified number.
+
+## Syntax
+
+```pseudo
+public static decimal Ceiling (decimal val);
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md:21]:
```suggestion
Math.Ceiling(val);
```

File: content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+public static double Ceiling(double val)
+```
+
+- The Ceiling method operates both functionalities in decimal and double.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md:25]:
```suggestion
- The method takes only one parameter, `val`, the number to be rounded (as a `decimal` or `double` type). 
```

File: content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+## Example 1
+
+The following example uses `Math.Ceiling(decimal)` to return the next full integer.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md:29]:
```suggestion
The following example uses `Math.Ceiling()` to return the next full integer.
```

File: content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+## Syntax
+
+```pseudo
+public static decimal Ceiling (decimal val);
+public static double Ceiling(double val)
+```
+
+- The Ceiling method operates both functionalities in decimal and double.
+
+## Example 1
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md:27]:
```suggestion
## Example
```

File: content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+}
+```
+
+The example will result in the following output:
+
+```shell
+Solution = 11
+Solution = 86
+Solution = -55
+```
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md:76]: I would recommend deleting the second example. For simple methods we generally only do one example unless there is some distinct functionality demonstrated by a second.

File: content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+}
+```
+
+The example will result in the following output:
+
+```shell
+Solution = 7
+Solution = -9
+```
+
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md:50]:
```suggestion
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@M3RF3R Finished the first review. Please tag me when you've made the updates.

**Comment by M3RF3R** [comment]
@caupolicandiaz done! Thanks for your corrections! I wasn't sure about the double and decimal approach but all the documentation I read explained the method like this.

**Review by caupolicandiaz — APPROVED** [review]
@M3RF3R Looks good, this is ready for another review.

File: content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
+---
+Title: '.Ceiling()'
+Description: 'Returns the smallest integer which is greater or equal to a given number.'
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/ceiling/ceiling.md:3]:
```suggestion
Description: 'Returns the smallest integer which is greater than or equal to a given number.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@M3RF3R Looks good, I only had one comment.

**Comment by M3RF3R** [comment]
@SSwiniarski I've commited the last changes but it appears that there are some errors

**Comment by SSwiniarski** [comment]
@caupolicandiaz @yangc95 @HishamT @KTom101 Any idea why the last test is failing?

**Comment by KTom101** [comment]
@M3RF3R @SSwiniarski `CatlogContent` in the metadata was written with a space like this -> `Catalog Content`. That was the error, I fixed it in commit [b79e708](https://github.com/Codecademy/docs/pull/1703/commits/b79e7087c04b2e4538165d3ab0b3ded4f4d8df92).

**Comment by M3RF3R** [comment]
@KTom101 Thanks a lot! I thought I had changed that!

**Review by SSwiniarski — APPROVED** [review]

**Comment by SSwiniarski** [comment]
@M3RF3R Congrats on the new entry! You can find it live here: https://www.codecademy.com/resources/docs/c-sharp/math-functions/ceiling
image

**Pull request merged**